dev/com.ibm.ws.transport.http/src/com/ibm/ws/genericbnf/internal/BNFHeadersImpl.java

```diff
@@ -1713,6 +1713,9 @@ private void clearAllHeaders() {
             elem.destroy();
             elem = next;
         }
+
+        //storage.clear();
+        //storage = new HashMap<Integer, HeaderElement>();
```

Comment:
```suggestion
```
Please don't leave in commented out code.

Reply:
Duh....